app/src/main/kotlin/com/yourssu/soomsil/usaint/screen/setting/SettingScreen.kt

```diff
@@ -228,16 +228,52 @@ fun SettingScreen(
     }
 }
 
-@Preview
+@Composable
+fun TitleText(
+    text : String
+) {
+    Box(
+        modifier = Modifier
+            .height(42.dp)
+            .padding(horizontal = 20.dp),
+        contentAlignment = Alignment.Center,
+    ) {
+        Text(
+            text = text,
+            fontWeight = FontWeight.Bold
+        )
+    }
+}
+
+@Composable
+fun ContentText(
+    text: String,
+    onClick: () -> Unit = {}
+) {
+    Box(
+        modifier = Modifier
+            .height(42.dp)
+            .padding(horizontal = 20.dp),
+        contentAlignment = Alignment.Center,
+    ) {
+        Text(
+            modifier = Modifier.clickable { onClick() },
+            text = text,
+            fontSize = 15.sp
```

Comment:
머티리얼의 typography로 지정해주세요!

```suggestion
            style = MaterialTheme.typography.<xxx>,
```

Reply:
아마 기존에 fontSize가 15sp였었어서 직접명시했었는데 typography중에서 fontSize가 15인게 없고 아예 뺴고 적용해보아도 이전과 동일하게 결과가 나와서 아예 뻈는데 어떤가요??
<img src="https://github.com/user-attachments/assets/942b3bc8-66f3-4db1-b98c-5382f4d2dbac" width="300" height="600">

커밋링크 : https://github.com/yourssu/Soomsil-USaint-Android/pull/50/commits/07a7707de13c203d95441bc6f76856beb3f455d1

Reply:
좋습니다! 어차피 디자인은 바뀔거라 지금은 봐줄만하기만 하면 돼요. 그냥 지정 안 했더니 되더라 이렇게 하고 끝내지 마시고, TextStyle 지정 안 해줬을 때 기본으로 어떤 스타일이 적용되는지 알아보시면 좋을 것 같아요!